### DISCUSSION TITLE
Una nota doverosa sulle scelte tecnologiche

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#18] [General]
L'ultimo messaggio di Nicola (@n1k9) citava OCaml e non è la prima volta che viene menzionato. Vorrei giustificare la scelta fatta.

Ho scelto OCaml per un solo motivo: è il linguaggio di programmazione che preferisco e uso da tantissimi anni e quando ho iniziato questo "progetto" l'ho fatto quasi per scherzo e per dimostrare che c'è chi prende sul serio le cazzate che si dicono dentro Joypad.

Per passare il tempo nei fine settimana scelgo qualcosa che almeno mi dia piacere, spero comprendiate. Poi potremmo discutere a lungo sui miei fine settimana, ma questo è un altro discorso.

<p align="center">
    <img width="200" height="300" src="https://user-images.githubusercontent.com/242049/168683865-b079dfc7-8af7-48fd-acd5-9adf6e9a05a3.jpg">
</p>

Se avessi anche solo immaginato che sarebbe diventato qualcosa di concreto con altre persone coinvolte avrei certamente scelto tecnologie più _mainstream_… e più inefficienti 😉 

Comunque che vi devo dire? È andata così e ora ve lo tenete, sto cammello 🐫

Come si dice a Roma: stacce!

**Comment by agofa** [comment]
> Come si dice a Roma: stacce!

Siccome sono molti, "statece" o "ce dovete stà"
Ecco.

**Comment by n1k9** [comment]
... ci sta tutto. 
Figurai io vorrei riprendere in mano il LISP (vecchie rimembranze universitarie).

**Reply by MarcoJul** [reply]
Direi che da stasera dovremmo inserire le parole Github e Ocaml nel database e fare il counting delle citazioni nelle punt/episodi😅

**Reply by pdonadeo** [reply]
Ho recuperato solo ora la live e ho capito il commento 😅

**Reply by agofa** [reply]
No spoiler ... :-) 
Lo sentirò stasera .....
Anzi grazie @pdonadeo che l'hai messo "tusaidove"...

**Comment by DGuidi** [comment]
Come ogni pet project, eventuali collaboratori dovrebbero essere contenti di poter imparare una tecnologia meno "comune"... Quale migliore occasione!
